Assets/Scripts/Controllers/Sprites/JobSpriteController.cs

```diff
@@ -122,7 +122,8 @@ protected override void OnCreated(Job job)
                 sr.sprite = usc.GetSpriteForUtility(job.JobObjectType);
             }
 
-            job_go.transform.position = job.tile.Vector3 + ImageUtils.SpritePivotOffset(sr.sprite);
+            job_go.transform.position = job.tile.Vector3 + ImageUtils.SpritePivotOffset(sr.sprite, BuildModeController.Instance.GetCurrentPreviewRotation());
```

Comment:
Should we only rotate furniture, and maybe use `job.buildablePrototype.Rotation` (if we only check for furniture)?